grails-app/domain/de/iteratec/osm/result/EventResult.groovy

```diff
@@ -95,6 +95,12 @@ class EventResult implements CsiValue, RepresentableWptResult {
      */
     Integer speedIndex
     Integer visuallyCompleteInMillisecs
+
+    Integer jsTotal
```

Comment:
It would be nice to include "bytes" in the name, otherwise the reader of this code has no idea what this actually refers to. So `jsTotalBytes`, `imageTotalBytes`, etc would be nice